CMakeLists.txt

```diff
@@ -23,6 +23,9 @@ if(BUILD_TESTING OR UNIFEX_BUILD_EXAMPLES)
   include(CTest)
 
   target_compile_features(gtest PUBLIC cxx_std_17)
+  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL GNU)
+    target_compile_options(gtest PRIVATE "-Wno-error=maybe-uninitialized")
+  endif()
```

Comment:
Why?

Reply:
GoogleTest raises this gcc warning (with gcc >= 10 only). And as all warnings are treated as errors, I had to disable this to be treated as an error.
This was unexpected in this PR, I'll remove this.